pkg/session/manager.go

```diff
@@ -91,3 +91,54 @@ func (m *Manager) authURI(provider oidc.Provider, state State, pkce pkce.PKCE) (
 
 	return u.String(), nil
 }
+
+func (m *Manager) StartTokenRefresher(ctx context.Context, interval time.Duration) {
+	go func() {
+		ticker := time.NewTicker(interval)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				if err := m.RefreshTokens(ctx); err != nil {
+					log.Printf("[ERROR] Failed to refresh tokens: %v", err)
+				}
+			case <-ctx.Done():
+				return
+			}
+		}
+	}()
+}
```

Comment:
Please make this function blocking. We can wrap it into `go func() {}()` any time.
With the current approach we can't track the flow whether the function finished execution or even started successfully.